lib/ast-converter.js

```diff
@@ -333,12 +404,21 @@ module.exports = function(ast, extra) {
             loc: getLoc(node, ast)
         };
 
+        /**
+         * Copies the result object
```

Comment:
Copies the result object into an ESTree  node with just a `type` property. This is used only for leaf nodes that have no other properties.